plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -22,19 +22,21 @@ class ExternalFeedDAO extends DAO {
 	/**
 	 * Constructor
 	 */
-	function __construct($parentPluginName) {
+	public function __construct($parentPluginName) {
 		$this->parentPluginName = $parentPluginName;
 		parent::__construct();
 	}
 
 	/**
 	 * Retrieve an ExternalFeed by ID.
 	 * @param $feedId int
+	 * @param $journalId int 
 	 * @return ExternalFeed
 	 */
-	function &getExternalFeed($feedId) {
+	function &getExternalFeed($feedId, $journalId) {
```

Comment:
References aren't necessary. Also, DAO getter conventions for fetching a single object should be `getById`, and the second parameter (`$journalId`) should be optional.